roles/openshift_node_dnsmasq/files/networkmanager/99-origin-dns.sh

```diff
@@ -111,6 +111,7 @@ EOF
           echo "# nameserver updated by /etc/NetworkManager/dispatcher.d/99-origin-dns.sh" >> ${NEW_RESOLV_CONF}
       fi
       sed -e '/^nameserver.*$/d' /etc/resolv.conf >> ${NEW_RESOLV_CONF}
+      echo "search svc.cluster.local cluster.local" >> ${NEW_RESOLV_CONF}
       echo "nameserver "${def_route_ip}"" >> ${NEW_RESOLV_CONF}
       if ! grep -q 'search.*cluster.local' ${NEW_RESOLV_CONF}; then
         sed -i '/^search/ s/$/ cluster.local/' ${NEW_RESOLV_CONF}
```

Comment:
Here already add the cluster.local search item, not sure why your host doesn't work

Reply:
#5398 will also solve my problem. And it's better than my PR. Thanks

Reply:
I would suggest do use a sed here also, just in case that a search is already defined in the resolv.conf.

```
if grep -q 'search .*' /etc/resolv.conf ; then
  sed -e 's/^search \(.*\)/search \1 svc.cluster.local cluster.local/" /etc/resolv.conf >> ${NEW_RESOLV_CONF}
else
echo "search svc.cluster.local cluster.local" >> ${NEW_RESOLV_CONF}
fi
```